components/centraldashboard-angular/frontend/src/app/pages/main-page/main-page.component.ts

```diff
@@ -28,17 +36,29 @@ export class MainPageComponent implements OnInit, OnDestroy {
   public get buildIdWithLabel() {
     return this.computeBuildValue(this.buildLabel, this.buildId);
   }
+  public menuLinks: menuLink[];
+  public externalLinks: any[];
+  public quickLinks: quickLink[];
```

Comment:
Why we don't render `externalLinks` and `quickLinks` anywhere? If we will have them in a following PR, fine by me.

Reply:
In the old dashboard, `quickLinks` and `documentationItems` are part of the `Home` page and this page will be a separate effort/PR. @kimwnasptd What about `externalLinks`?

Reply:
@orfeas-k There is no need to wait. If we want the `externalLinks` we can add them in the next PR.